providers/os/provider/provider_test.go

```diff
@@ -106,8 +104,8 @@ func TestLocalConnectionIdDetectors_DelayedDiscovery(t *testing.T) {
 	require.Contains(t, connectResp.Asset.IdDetector, ids.IdDetector_Hostname)
 	require.Contains(t, connectResp.Asset.IdDetector, ids.IdDetector_CloudDetect)
 	require.NotContains(t, connectResp.Asset.IdDetector, ids.IdDetector_SshHostkey)
-	// Now the platformIDs are cleaned up
-	require.Len(t, connectResp.Asset.PlatformIds, 2)
```

Comment:
This list had a thing that wasn't a platform id in it. It was just the hostname directly